BMI Calculator/BMI Calculator/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Dynamic;
+
+namespace BMI_Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            UserInfo();
+            UserInfo();
+        }
+
+        static void CalBMI(double weight, double height, string fName, string lName)
+        {
+            double bmi = (weight / (height * height)) * 10000;
+            Console.WriteLine("{0} {1}, your BMI is: {2:F2}", fName, lName, bmi);
+        }
+
+        static void PrintUserInfo(string fName, string lName, double age, double weight, double height)
+        {
+            string userInfo = $"{fName} {lName} is {age} years old, his weight is {weight} kg and his height is {height} cm";
+            Console.WriteLine(userInfo);
+
+            CalBMI(weight, height, fName, lName);
+        }
+
+        static void UserInfo()
```

Comment:
Method names should at the very least contain a verb. 
What is the method going to do? Perhaps `QueryUserInfo()` or `PromptUserInfo()`?